rmf_site_format/src/sdf.rs

```diff
@@ -870,6 +877,8 @@ mod tests {
             ..Default::default()
         };
         let s = yaserde::ser::to_string_with_config(&sdf, &config).unwrap();
-        std::fs::write("test.sdf", s).unwrap();
+        if let Err(e) = std::fs::write("test.sdf", s) {
+            error!("Unable to write test.sdf {:?}", e);
```

Comment:
This is inside of a test so we actually do want to use `.unwrap()` here. We want the tests to exit with an error code right away whenever something violates our expectations.